scripts/generate-config.ts

```diff
@@ -13,12 +13,15 @@ const getRules = (
   predicate: (rule: RuleModule) => boolean,
   setting = (rule: RuleModule) => rule.meta.docs?.recommended || 'warn',
 ) =>
-  Object.entries(rules).reduce((rules, [ruleName, rule]) => {
-    if (predicate(rule)) {
-      rules[`${RULE_NAME_PREFIX}${ruleName}`] = setting(rule)
-    }
-    return rules
-  }, {} as Record<string, string>)
+  Object.entries(rules).reduce<Record<string, string>>(
```

Comment:
I'm not a big fan of using generics to be honest 😅
I prefer casting it myself, because generics can change and have breaking changes.
It probably won't be the cause with `.reducer` though.

Reply:
Oh, I prefer the opposite, but in that case I can revert these changes if you want to. 😄

Reply:
Nah, it's fine.